internal/cli/login/login.go

```diff
@@ -37,7 +37,7 @@ const (
 	defaultRandStringCharSet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 )
 
-func NewCommand(opt *option.Option) *cobra.Command {
+func NewCommand(_ *option.Option) *cobra.Command {
```

Comment:
Nit for future reference -- not something that requires correcting here. If you're not using _any_ of the params you receive, you can just not name them at all instead of using `_`.